assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -90,38 +113,42 @@ function Header({
     }
 
     return (
-      <HeaderToggleButtonContainer>
-        <ToggleButtonGroup
-          buttons={STORY_STATUSES.map((storyStatus) => {
-            if (
-              storyStatus.status === STORY_STATUS.PRIVATE &&
-              (!totalStoriesByStatus.private ||
-                totalStoriesByStatus.private < 1 ||
-                !canReadPrivatePosts)
-            ) {
-              return null;
-            }
-            return {
-              handleClick: () => {
+      <>
+        {STORY_STATUSES.map((storyStatus) => {
+          if (
+            storyStatus.status === STORY_STATUS.PRIVATE &&
+            (!totalStoriesByStatus.private ||
```

Comment:
let's make this check a positive boolean and not mix the logical operators in the conditional statement
```
const isStoryPrivate = storyStatus.status === STORY_STATUS.PRIVATE
// I think the following line can be updated to just be:
// !totalStoriesByStatus.private || !canReadPrivatePosts
// first part evaluates to true when totalStoriesByStatus.private is -1, 0, or undefined
const canReadPrivate = !totalStoriesByStatus.private || totalStoriesByStatus.private < 1 || !canReadPrivatePosts
if (isStoryPrivate && canReadPrivate) { ... }
```

Reply:
yeah! thanks! completely looked over this as a potential area to get updated.